source/includes/_overview.md

```diff
@@ -50,5 +53,7 @@ You can search and filter objects returned by an endpoint using URL parameters.
 For searching, you need to include the `search` parameter in the URL. e.g.
 `/api/v1/households/?search=Smith`
 
-For filtering, you need to include the desired attribute in the URL parameters. Each section describes the allowed **Filtering Fields** for that endpoint. For instance, if you want to list the accounts of the investor with ID 31 you can use
+For filtering, you need to include the desired attribute in the URL parameters. Each section describes the allowed *
+*Filtering Fields** for that endpoint. For instance, if you want to list the accounts of the investor with ID 31 you can
```

Comment:
Here there could be an error in the syntax because the double asterisk (`**`) is separated, preventing the bold style from being applied.

Reply:
hey wolf! very nice catch there... indeed it would generate a problem